Scala/Scala.sublime-syntax

```diff
@@ -138,7 +138,7 @@ contexts:
           pop: true
         - match: '([a-zA-Z$_][a-zA-Z0-9$_]*)\s*:\s*'
           captures:
-            1: entity.name.variable variable.parameter
+            1: entity.name.parameter variable.parameter
```

Comment:
Ok, so let's not index function parameters then.  The slightly-weird part about that though is the indexer *will* consider variables in pattern matching or for-comprehensions (once #491 is merged), and those elements are highlighted identically to function parameters.  In practice, indexing these pattern variables has been enormously useful (indexing function parameters hasn't been quite as vital), so I'd definitely like to keep that functionality.